config/locales/en/providers.yml

```diff
@@ -239,3 +240,17 @@ en:
     success_prospects:
       show:
         h1-heading: What are the prospects of success?
+    limitations:
+      show:
+        h1-heading: What you're covered for
+        h2-heading: Proceeding
+        body: As to proceedings under Part IV Family Law Act 1996 limited to all steps up to and including obtaining and serving a final order and in the event of breach leading to the exercise of a power of arrest to representation on the consideration 	of the breach by the court (but excluding applying for a warrant of arrest, if not attached, and representation in contempt proceedings).
```

Comment:
Break this over multiple lines. If you use the YAML `>` operator the line breaks will be removed when rendered in the HTML markup. Personally I'd be tempted to use `|` as then the text will be more readable in the markup as well - it line breaks will disappear when rendered in the browesr. So:
```yaml
body: >
  As to proceedings under Part IV Family Law Act 1996 limited to all steps up to and
  including obtaining and serving a final order and in the event of breach leading to 
  the exercise of a power of arrest to representation on the consideration of the 
  breach by the court (but excluding applying for a warrant of arrest, if not attached, 
  and representation in contempt proceedings).
```
See https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines